src/pages/Wallet/AddressesPage.tsx

```diff
@@ -92,102 +81,86 @@ const AddressesPage = () => {
-            buttonText={t`Start`}
-            onButtonClick={() => setIsConsolidationModalOpen(true)}
-            infoLink={links.utxoConsolidation}
-          />
-          <OperationBox
-            title={t`Generate one address per group`}
-            Icon={<HardHat color="#a880ff" strokeWidth={1} size={55} />}
-            description={t`Useful for miners or DeFi use.`}
-            buttonText={isPassphraseUsed ? t`Generate` : t`Start`}
-            onButtonClick={handleOneAddressPerGroupClick}
-            infoLink={links.miningWallet}
-          />
-          <OperationBox
-            title={t`Discover active addresses`}
-            Icon={<Search color={theme.global.complementary} strokeWidth={1} size={55} />}
-            description={t`Scan the blockchain for addresses you used in the past.`}
-            buttonText={t`Search`}
-            onButtonClick={() => discoverAndSaveActiveAddresses(activeWalletMnemonic)}
-            infoLink={links.miningWallet}
-          />
-          <OperationBox
-            placeholder
-            title={t`More to come...`}
-            Icon={<Lightbulb color={theme.font.secondary} strokeWidth={1} size={28} />}
-            description={t`You have great ideas you want to share?`}
-            buttonText={t`Tell us!`}
-            onButtonClick={() => openInWebBrowser(links.discord)}
+      <MainPanel>
+        <Header>
+          <div>
+            <Title>{t`Addresses & contacts`}</Title>
+            <Subtitle>{t`Easily organize your addresses and your contacts for a more serene transfer experience. Sync with the mobile wallet to be more organized on the go.`}</Subtitle>
+          </div>
+          <div>
+            <AnimatePresence>
+              {!infoMessageClosed && (
+                <InfoMessage link={links.faq} onClose={closeInfoMessage}>
+                  {t`Want to know more? Click here to take a look at our FAQ!`}
+                </InfoMessage>
+              )}
+            </AnimatePresence>
+          </div>
+        </Header>
+        <Tabs>
+          <TabBar items={tabs} onTabChange={(tab) => setCurrentTab(tab)} activeTab={currentTab} />
+          <TabContent>{tabs.find((t) => t.value === currentTab.value)?.component}</TabContent>
+        </Tabs>
+      </MainPanel>
+      <AdvancedOperationsPanel>
+        <AdvancedOperationsHeader>
+          <AdvancedOperationsTitle>{t`Advanced operations`}</AdvancedOperationsTitle>
+          <AdvancedOperationsToggle
+            label={t`Show advanced operations`}
+            description={t`Open the advanced feature panel.`}
+            InputComponent={
+              <Toggle
+                label={t`Show advanced operations`}
+                toggled={isAdvancedSectionOpen}
+                onToggle={() => setIsAdvancedSectionOpen(!isAdvancedSectionOpen)}
+              />
+            }
           />
-        </AdvancedOperations>
-      </ExpandableSection>
+        </AdvancedOperationsHeader>
+        <Collapsable
```

Comment:
Ah, interesting idea! So basically it would feel like a bottom modal. If this row would be stick'ed to the bottom wouldn't that limit the available vertical space of the "Addresses" tab? Shall we also try the Ctrl+Scroll idea in there as well?

Another clarification: Would the "Advanced operations" row with the toggle button remain at the bottom when activated and only the operation boxes would appear on top? Or would the whole row move up? I think I find the first option better because I don't have to move the mouse to toggle it off. WDYT?

Reply:
Here's what I could come up with! I am not 100% happy with the bottom modal because I need to define its content height. I couldn't yet find a way to have dynamic modal height based on the content height. I would like it to move on, however, since we are still experimenting with this. Here's a working prototype:

https://user-images.githubusercontent.com/1579899/211323385-a29fec28-4df0-4e0b-9f0e-2397ab61c4fd.mp4

Reply:
That's exactly what I had in mind, and your implementation looks perfect for now. Let's see if we want to expand this to other pages in the future. The height may need to be computed manually.